src/busctl/busctl.c

```diff
@@ -1002,6 +1001,8 @@ static int introspect(int argc, char **argv, void *userdata) {
         if (r < 0)
                 return bus_log_parse_error(r);
 
+        (void) pager_open(arg_pager_flags);
```

Comment:
hmm, this means we start the pager a long time before we actually start outputting anything, and before the bus calls that possibly will fail. I think it would be nicer to leave the page_open() where it is in this function, and just insert a second call in the if block right below, so that we use it only when we are ready to output stuff, but not before